packages/core/src/demo/echo.js

```diff
@@ -41,6 +41,18 @@ const messageEchoCache = {};
   const apiClient = new APIClient({store: engine, urls: backend});
   const account = new Account(apiClient);
 
+  const sendConfirmation = async (messageId, conversationId, messageTimer, userId) => {
+    const confirmationPayload = account.service.conversation.createConfirmation(messageId);
+    await account.service.conversation.send(conversationId, confirmationPayload);
+
+    if (messageTimer && userId) {
+      setTimeout(
+        () => account.service.conversation.deleteMessageEveryone(conversationId, messageId, [userId]),
+        messageTimer
+      );
```

Comment:
Still don't get why we would send this confirmation on messageTimer timeout even in the demo case.

Reply:
But we send a confirmation in [line 46](https://github.com/wireapp/wire-web-packages/pull/939/files#diff-d497294ce653e28d25a71b3bf9dfb025R46).